icode.properties

```diff
@@ -1,12 +1,18 @@
 category=External Analyzers
-description=Import i-Code CNES results for Fortran(77 & 90) and Shell
+description=Analyze your Fortran(77 & 90) and Shell code with i-Code CNES
 homepageUrl=https://github.com/lequal/sonar-icode-cnes-plugin
 archivedVersions=1.0.2,1.1.0
-publicVersions=
+publicVersions=2.0.0
 
 defaults.mavenGroupId=fr.cnes.sonarqube.plugins
 defaults.mavenArtifactId=sonaricode
 
+2.0.0.description=i-Code CNES is now embedded in the plugin, no more need to install it
```

Comment:
Just to be super-clear: i-Code CNES is free and open source? Since you're changing the paradigm here I want to be really transparent on that before we move forward with this version.

Reply:
Yes i-Code CNES is totally free and open source, our organization distributes it here: https://github.com/lequal/i-CodeCNES